src/components/Articles.jsx

```diff
@@ -0,0 +1,19 @@
+const Article = ({ArticleData,index}) => {
+    return (<>
+    <li className="article-item">
+        <a href={`/article/${ArticleData.article_id}`}>
+            <img src={ArticleData.article_img_url } alt="new articles image" />
```

Comment:
alt tags don't need the word image or picture in them, it's much better to be matter of fact with alt text they're here to give context to screen readers